public/tier1/convar.h

```diff
@@ -1,4 +1,4 @@
-//===== Copyright � 1996-2005, Valve Corporation, All rights reserved. ======//
+//===== Copyright � 1996-2005, Valve Corporation, All rights reserved. ======//
 //

```

Comment:
The new copyright symbol looks like a diamond with a question mark on my system (windows 10)

Reply:
Would you plz modify © to (C)?  Very thanks if you could make this modification. Some system unable to display this symbol. (Like in this pr it display to �)

Reply:
This is a draft PR, and not ready for review. Be patient thank you..